apiary.apib

```diff
@@ -762,6 +762,39 @@ Get a list of all work types, sorted alphabetically (on their name).
                 + id: `811a5825-96f4-4318-83c3-2840935c6003` (string)
                 + name: `Planning` (string)
 
+## Templates [/templates]
+
+### templates.list [POST /templates.list]
+
+Get a list of all templates.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + department_id: `a344c251-2494-0013-b433-ccee8e8435e5` (string)
+            + type: `invoice` (enum[string])
```

Comment:
PP: add `optional` to both optional filters.

Reply:
Doublechecking with Charlotte if these are optional. Otherwise, I'll need to add the required param

Reply:
Changes are implemented